agent/pom.xml

```diff
@@ -16,13 +17,11 @@
         <dependency>
             <groupId>com.zegelin.cassandra-exporter</groupId>
             <artifactId>common</artifactId>
-            <version>0.9.7-SNAPSHOT</version>
```

Comment:
Does `mvn release:prepare` work if this is moved to a property?

Reply:
good catch, i put that version back in parent's dep. management without property resolution